pwiz_tools/Skyline/Util/Util.cs

```diff
@@ -1918,6 +1919,17 @@ private static void ReportExceptionForRetry(int milliseconds, Exception x, int l
         // in case of dotCover and dotMemory."
         public static bool RunningResharperAnalysis => !string.IsNullOrEmpty(Environment.GetEnvironmentVariable(@"JETBRAINS_DPA_AGENT_ENABLE"));
 
+        [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]

```

Comment:
Do we really not build any differently for the Wine Docker container version? It is really just the 64-bit binaries running on Wine? I was expecting this to be something controlled at build time. Pretty cool, if the binaries are not different in any way. But, running on Wine is still useful information, since there have been cases where the code works slightly differently on Wine... Though, it does raise the question whether we should just make it a system version value rather than part of the software version text.